cvat/apps/engine/task.py

```diff
@@ -341,6 +366,10 @@ def _create_thread(tid, data):
     upload_dir = db_task.get_upload_dirname()
     video, archive = _validate_data(data)
 
+    # parse step filter
+    match = re.search("step *= *([0-9]+)", db_task.frame_filter)
```

Comment:
I don't like that a wrong filter value can be accepted. Also if you want to allow spaces around "=" use `\s*` instead. Instead of `[0-9]+` I would use `[1-9]\d*`

Reply:
I suggest writing a validator for frame_filter and check value explicitly: https://www.django-rest-framework.org/api-guide/serializers/#field-level-validation

Reply:
Updated the re, and also added a field validation method.